tests/characterization_data/filters_expected/sample_nsxv.nsx

```diff
@@ -1,54 +1,612 @@
-<!--
- $Id:$
- $Date:$
- $Revision:$
--->
+<?xml version="1.0" ?>
```

Comment:
@ankenyr a nit to the best of my knowledge (in that I didn't try to load it to see what would happen).

I'll load it into NSX and see if it gets returned in the same way.